wallet/subnet/primary/examples/add-permissioned-subnet-validator/main.go

```diff
@@ -60,9 +60,13 @@ func main() {
 	pWallet := wallet.P()
 
 	addValidatorStartTime := time.Now()
+	shortNodeID, err := ids.ShortNodeIDFromNodeID(nodeID)
+	if err != nil {
+		log.Fatalf("failed turning NodeID %v into shortNodeID: %s\n", nodeID, err)
```

Comment:
I feel like we should be consistent with the capitalization
```suggestion
		log.Fatalf("failed turning NodeID %s into ShortNodeID: %s\n", nodeID, err)
```

Reply:
Done